README.md

```diff
@@ -352,14 +349,25 @@ records and that you are using the client code that is not dependent on the
 local credentials file.
   
 ##### Command to run Job on AI Platform (final version after testing):
+First, make sure you are running in the correct project if you have multiple:
+```
+gcloud config set project <project-id>
+```
+
 **Note:** See deploy.py code for a python script that simplifies the process of  
 running the gcloud commands for deploying jobs and prediction routines. 
 
-Sample running via deploy.py:
+**Sample running via deploy.py:**
 ```
-python deploy.py train --bucket <your_bucket> --name <name_of_training_job>
+python deploy.py train 
+  --staging-path <gs://staging_bucket/where_to_store_training_package>
```

Comment:
Looks like we do not need --staging-path anymore. When I tried running with the staging-path, it would not run, but when I took it out, it ran successfully. When I did -help command, it looks like this is no longer a needed argument. 
PS C:\git\end-to-end-model-test-gcp> python deploy.py train -h
usage: deploy train [-h] --bucket BUCKET --path PATH --name NAME

optional arguments:
  -h, --help       show this help message and exit
  --bucket BUCKET  Specify the storage bucket for saving model related output
  --path PATH      Specify the path in bucket for saving model related output
  --name NAME      Specify name of job for AI Platform Jobs

Everything else looks great! Once that is changed, let me know and I will approve!

Reply:
Good catch! Fixed it.